colorbleed/plugins/maya/publish/submit_deadline.py

```diff
@@ -160,6 +160,9 @@ def process(self, instance):
                 # Only render layers are considered renderable in this pipeline
                 "UsingRenderLayers": True,
```

Comment:
> UsingRenderLayers

I didn't see this one in the plug-in submission data of Deadline itself - what is this supposed to do?

Reply:
Had a quick look, it is being used. In our case it isn't actually doing anything because it's always used like this:

```
if self.Renderer == "file" or ( self.UsingRenderLayers and len( self.RenderLayer ) == 0 ):
```

And our renderer is always set to to `"file"` in our submissions - but for now just let's keep the code in. :)